src/_pytest/pathlib.py

```diff
@@ -100,10 +100,24 @@ def chmod_rw(p: str) -> None:
     return True
 
 
+def create_long_path(path: Path) -> Path:
```

Comment:
Looking again, I think the function name might be a little confusing, in that it might imply that the path is actually created.

From the link, the formal name for this seems to be "extended length path". So how about `ensure_extended_length_path`?